src/main/java/com/elixr/poc/bulkimport/dto/Patient.java

```diff
@@ -0,0 +1,23 @@
+package com.elixr.poc.bulkimport.dto;
+
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@Document(collection = "patients")
+public class Patient {
+    @Id
+    private String patientId;
+    private String patientFirstName;
+    private String patientLastName;
+    private int patientAge;
+    private String patientAddress;
+    private List<String> doctorId;
```

Comment:
Name this field as doctorList. When you say doctor id , it seems to indicate a single element and not the list

Reply:
Updated.